### PR TITLE
i18n: Add Polish translation

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

**Comment by mkljczk** [comment]
It would be nice if my name will link to my GitHub XD

File: static/js/global.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ function updateTranslation(localex) {
 $(window).on('load', function() {
   var localeOptions = {
     English: 'en',
-    Español: 'es',
+    Español: 'es'
```
Comment (andrewda) [inline static/js/global.js]: You'll need a comma here, this will crash as-is.

**Review by andrewda — CHANGES_REQUESTED** [review]

**Comment by andrewda** [comment]
@m4sk1n if you join the coala org and make yourself a public member it'll find you ;)

**Comment by andrewda** [comment]
Nevermind, just realized you're a part of Ubuntu, not coala.

File: static/js/global.js (hunk 1 above)
Comment (mkljczk) [reply static/js/global.js]: `vi` started to behave really strange on my phone, sorry for that -_-

Comment (jayvdb) [reply static/js/global.js]: You submitted a PR from your `master` branch :P

Comment (mkljczk) [reply static/js/global.js]: I know, @jayvdb… I don’t need to create branches, because I’m not going to change anything more in this repo…

**Comment by mkljczk** [comment]
@andrewda but some of leaders’ names outside coala are also linking to their GitHub profiles…

**Comment by andrewda** [comment]
Yea – so the way it's currently being done is checking if the user is either A.) in the organization's GitHub org or B.) has made commits/contributions to a repository in the GitHub org. I don't think Ubuntu primarily uses GitHub, so I'm guessing that's the issue. If you're on the coala Gitter, we could move this conversation there so as not to clutter this PR.

**Comment by mkljczk** [comment]
Ok, joined to Gitter channels, I don’t know how to properly mention you from matrix.org… Most of Ubuntu-related tasks on GitHub are happening in the canonical-websites/tutorials.ubuntu.com repo.
Also, I think it’s fully ready for merge

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/96/commits/5174e2f8ad82f40a9a894f13024249067d5a10ea

**Review by andrewda — APPROVED** [review]

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
Merged! Thank you.

**Pull request merged**